gadgets/mri_core/AutoScaleGadget.h

```diff
@@ -1,35 +1,25 @@
-#ifndef AUTOSCALEGADGET_H_
-#define AUTOSCALEGADGET_H_
+/**
+    \brief  Autoscales real-type images based on a given max value and the 99th percentile of the data
+    \author Original: David Christoffer Hansen
```

Comment:
I think technically I wrote the original AutoScaleGadget (in 2011), but I am not sure if we should even keep peoples names in here?

Reply:
I like killing names across the board, it's all in git history anyway